support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -67,26 +61,24 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
 
 class ZuoraSubscriptionCreator(
   now: () => DateTime,
-  promotionService: PromotionService,
-  redemptionService: DynamoLookup with DynamoUpdate,
   zuoraService: ZuoraSubscribeService,
-  giftCodeGenerator: GiftCodeGeneratorService,
-  config: ZuoraConfig,
+  subscriptionDataBuilder: SubscriptionBuilder,
```

Comment:
```suggestion
  subscriptionDataBuilder: SubscriptionDataBuilder,
```

I'd already put this suggestion below but for consistency...